runtime/autoload/man.vim

```diff
@@ -71,16 +65,56 @@ function! man#read_page(ref) abort
   call s:read_page(path)
 endfunction
 
+" Handler for s:system() function.
+function! s:system_handler(jobid, data, event) abort
+  if a:event == 'stdout'
+    let self.output .= join(a:data, "\n")
+  elseif a:event == 'exit'
+    let s:shell_error = a:data
+  endif
+endfunction
+
+" Run a system command and timeout after 30 seconds.
+function! s:system(cmd, ...) abort
+  let stdin = a:0 ? a:1 : ''
+  let opts = {
+        \ 'output': '',
+        \ 'on_stdout': function('s:system_handler'),
+        \ 'on_exit': function('s:system_handler'),
+        \ }
+  let jobid = jobstart(a:cmd, opts)
+
+  if jobid < 1
+    throw printf('Command error %d: %s', jobid, 
```

Comment:
Just for consistency's sake, "Command" should not be capitalized in this error and the ones below.